lib/blockchain/chain.js

```diff
@@ -3223,7 +3223,7 @@ class Chain extends AsyncEmitter {
       const state = await this.getState(prev, deployment);
 
       if (state === thresholdStates.LOCKED_IN
-          || state === thresholdStates.STARTED) {
+          || (state === thresholdStates.STARTED && deployment.force)) {
```

Comment:
Is this effectively the same fix we did here?

https://github.com/handshake-org/hsd/commit/5e20a5aaaa7d4254ba88b14acd3b85f025330029